#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_vdr_1.3.32-fix.dpatch by Thomas Schmidt <tschmidt@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix building for vdr 1.3.32

@DPATCH@
diff -urNad vdr-plugin-streamdev-0.3.1+cvs20050522/client/filter.c /tmp/dpep.TvsJ2Z/vdr-plugin-streamdev-0.3.1+cvs20050522/client/filter.c
--- vdr-plugin-streamdev-0.3.1+cvs20050522/client/filter.c	2005-09-11 21:49:11.357677496 +0200
+++ /tmp/dpep.TvsJ2Z/vdr-plugin-streamdev-0.3.1+cvs20050522/client/filter.c	2005-09-11 21:49:12.288535984 +0200
@@ -8,6 +8,7 @@
 #include "common.h"
 
 #include <vdr/ringbuffer.h>
+#include <vdr/device.h>
 
 #if VDRVERSNUM >= 10300
 
diff -urNad vdr-plugin-streamdev-0.3.1+cvs20050522/remux/ts2es.h /tmp/dpep.TvsJ2Z/vdr-plugin-streamdev-0.3.1+cvs20050522/remux/ts2es.h
--- vdr-plugin-streamdev-0.3.1+cvs20050522/remux/ts2es.h	2005-05-22 00:29:37.000000000 +0200
+++ /tmp/dpep.TvsJ2Z/vdr-plugin-streamdev-0.3.1+cvs20050522/remux/ts2es.h	2005-09-11 21:49:55.368986760 +0200
@@ -3,6 +3,7 @@
 
 #include "remux/tsremux.h"
 #include <vdr/ringbuffer.h>
+#include <vdr/device.h>
 
 class cTS2ES;
 class cRingBufferLinear;
diff -urNad vdr-plugin-streamdev-0.3.1+cvs20050522/remux/ts2ps.c /tmp/dpep.TvsJ2Z/vdr-plugin-streamdev-0.3.1+cvs20050522/remux/ts2ps.c
--- vdr-plugin-streamdev-0.3.1+cvs20050522/remux/ts2ps.c	2005-05-22 00:29:37.000000000 +0200
+++ /tmp/dpep.TvsJ2Z/vdr-plugin-streamdev-0.3.1+cvs20050522/remux/ts2ps.c	2005-09-11 21:49:38.905489592 +0200
@@ -1,6 +1,7 @@
 #include "remux/ts2ps.h"
 #include "server/streamer.h"
 #include <vdr/channels.h>
+#include <vdr/device.h>
 
 class cTS2PS {
 	friend void PutPES(uint8_t *Buffer, int Size, void *Data);
